docs/conf.py

```diff
@@ -19,7 +19,11 @@
 # The short X.Y version
 version = ""
 # The full version, including alpha/beta/rc tags
-release = "1.15.3"
+release = ""
+
+with open(os.path.join("../atlassian", "VERSION")) as file:
```

Comment:
This relies on the working directory of the shell. To be independant from the working directory the variable __file__ should be used. But at the moment there are more lines which relies on the working directory.